projects/WTI-API/packageWTI.xml

```diff
@@ -2,6 +2,12 @@
 
 <project name="package-WTI" default="archives" basedir=".">
 	
+    <property environment="env"/>
+    <condition property="exists.GITHUB_RUN_NUMBER">
+        <isset property="env.GITHUB_RUN_NUMBER"/>
+    </condition>
+    <property name="lastGITLABbuild" value="6259" />
+
```

Comment:
Once again:  it would be nice if there was at least a line or two of comments explaining why this is being added to the `packageWTI.xml` file...